src/command_modules/azure-cli-storage/tests/test_storage_validators.py

```diff
@@ -10,8 +10,10 @@
                                                            get_datetime_type, datetime, ipv4_range_type, resource_type_type,
                                                            services_type, process_blob_source_uri)
 from azure.cli.core.profiles import get_sdk, ResourceType
+from azure.cli.testsdk import profile_version_constraint
 
 
+@profile_version_constraint(min_api='latest')
```

Comment:
`latest` should be an alias for the latest available profile version. It shouldn't replace the real name of the profile. Otherwise, when a new profile is introduced all the tests marked with `latest` need to be updated.